### PR TITLE
Fix fault injection tests

### PR DESCRIPTION
**Pull request by mmichal10**

### CODE REVIEW

File: test/functional/tests/fault_injection/test_soft_hot_plug_device.py [hunk 1]
```diff
@@ -10,7 +10,7 @@
 import pytest
 
 from api.cas import casadm
-from api.cas.cache_config import CacheMode
```
Comment (robertbaldyga) [inline test/functional/tests/fault_injection/test_soft_hot_plug_device.py:13]: It doesn't look like `CacheStatus` is used anywhere in this test.

File: test/functional/tests/fault_injection/test_soft_hot_plug_device.py [hunk 2]
```diff
@@ -50,6 +50,8 @@ def test_soft_hot_plug_cache(cache_mode):
 
     with TestRun.step("Soft hot unplug cache device"):
         cache_dev.unplug()
+        if cache_dev.is_detected():
+            TestRun.block("Failed to unplug the device!")
```
Comment (robertbaldyga) [inline test/functional/tests/fault_injection/test_soft_hot_plug_device.py:54]: This check should be performed by the TF itself.

File: test/functional/api/cas/statistics.py [hunk 3]
```diff
@@ -52,7 +52,7 @@ def __init__(self, stats):
             self.config_stats = CacheConfigStats(
                 *[stats[stat] for stat in config_stats_cache]
             )
-        except KeyError:
+        except KeyError as e:
```
Comment (robertbaldyga) [inline test/functional/api/cas/statistics.py:55]: This doesn't look necessary. `e` is not used.

**Review by robertbaldyga — CHANGES_REQUESTED** [review]

File: test/functional/api/cas/statistics.py (hunk 3 above)
Comment (Deixx) [inline test/functional/api/cas/statistics.py:55]: e not used

File: test/functional/tests/fault_injection/test_soft_hot_plug_device.py [hunk 4]
```diff
@@ -10,7 +10,7 @@
 import pytest
 
 from api.cas import casadm
-from api.cas.cache_config import CacheMode
+from api.cas.cache_config import CacheMode, CacheStatus
```
Comment (Deixx) [inline test/functional/tests/fault_injection/test_soft_hot_plug_device.py:13]: CacheStatus not used

**Review by Deixx — CHANGES_REQUESTED** [review]

File: test/functional/tests/fault_injection/test_soft_hot_plug_device.py (hunk 2 above)
Comment (mmichal10) [reply test/functional/tests/fault_injection/test_soft_hot_plug_device.py:54]: Actually it is:
image
But if invalid serial has been set for a disk, TF reports the disk as already unplugged. After reconsidering the issue I will add a check to the function responsible for parsing configuration file.

**Pull request closed without merging**